packages/graph/hash_graph/lib/graph/src/api/rest/link.rs

```diff
@@ -48,9 +48,9 @@ impl RoutedResource for LinkResource {
                 "/entities/:entity_id/links",
                 post(create_link::<P>)
                     .get(get_entity_links::<P>)
-                    .delete(inactivate_link::<P>),
+                    .delete(remove_link::<P>),
             )
-            .route("/links", get(get_active_links::<P>))
+            .route("/links", get(get_current_links::<P>))
```

Comment:
Do we use "current" anywhere else? Like get_current_data_types or anything? I just don't see the need for describing a link as "current". Isn't it just a link? And then it gets deleted?

Reply:
Good point, will just call it `get_links` instead